### DISCUSSION TITLE
Auto convert to a foreign currency

### DISCUSSION THREAD

**Discussion by Zubayer204** [firefly-iii/firefly-iii#8417] [Firefly III] [answered]
As someone outside from the US, I have to make a lot of transactions both in my home currency as well as US Dollar. Is there a custom action which can automatically convert transactions with US dollar to my native currency and keep both of the currencies in that transaction?

**Comment by JC5** [comment, marked as answer]
No. You can add something manually using webhooks. Most users import transactions from their bank and rely on the bank's conversion.

**Reply by Zubayer204** [reply]
Thank you for the clarification. I'll use webhooks and api to listen to transaction creation and update according.
